metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/alerts_ui_master.py

```diff
@@ -40,10 +41,9 @@ def install(self, env):
         self.install_packages(env)
 
     def configure(self, env, upgrade_type=None, config_dir=None):
-        print 'configure alerts_ui'
         from params import params
         env.set_params(params)
-
+        metron_service.refresh_configs(params)
```

Comment:
If we move the property to metron-rest-env then I'm not sure this is necessary.

Reply:
The call to `refresh_configs` is the only thing that pushes changes to the global config.  I don't see how the setting would be pushed to the global config without it.  Let me know if I am misunderstanding something.

Reply:
You are correct, of course.  Doh.  I'll drop this.